### ISSUE TITLE
Json log format not working since v1.44.0 (ECS_LOG_OUTPUT_FORMAT)

### ISSUE THREAD

**Issue by adebasi** [aws/amazon-ecs-agent#2591]
### Summary
The output format json, configured via ECS_LOG_OUTPUT_FORMAT, doesn't have any effect from version 1.44.0 on. 

### Description
In version 1.36.0 the output format json was introduced. Users were able to set the format to json by setting the env var ECS_LOG_OUTPUT_FORMAT. From 1.44.0 this seems to be broken and I was not able to find something related to this in the changelog. 

I verified this behaviour by running containers with different versions:
```
$ docker run -e ECS_LOG_OUTPUT_FORMAT=json amazon/amazon-ecs-agent:v1.43.0
{"level": "info", "time": "2020-08-27T07:33:10Z", "msg": "Loading configuration", "module": "agent.go"}
```
```
$ docker run -e ECS_LOG_OUTPUT_FORMAT=json amazon/amazon-ecs-agent:v1.44.0
1598513604929243900 [Error] node must have children
1598513604932176700 [Info] Loading configuration
``` 

I noticed this error because I updated our agents to version 1.44.1 and setting ECS_LOG_OUTPUT_FORMAT to json had no effect.

### Expected Behavior
Set ECS_LOG_OUTPUT_FORMAT to json has the effect that the container logs are in json format.

### Observed Behavior
see description

**Comment by ubhattacharjya** [comment]
Hi @adebasi ,

I am so sorry to hear that you are facing this problem. I am investigating the issue from my end. I will update you with my findings.

Utsa

**Comment by ubhattacharjya** [comment]
Hi @adebasi ,

I could reproduce the error that you faced.

Kindly use the following:

docker run -e ECS_LOG_OUTPUT_FORMAT=json -e ECS_LOGFILE=/log/ecs-agent.log amazon/amazon-ecs-agent:v1.44.0

Let me know if you are still facing the issue

Utsa

**Comment by ubhattacharjya** [comment]
The PR for fixing this issue : https://github.com/aws/amazon-ecs-agent/pull/2598/files

**Comment by adebasi** [comment]
Hi @ubhattacharjya,
Thank you for the quick response and fixing the bug.

**Comment by ellenthsu** [comment]
This fix has been released as part of ECS Agent 1.44.4: https://github.com/aws/amazon-ecs-agent/releases/tag/1.44.4

Please perform an update of the Agent: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-agent-update.html or you can find the latest ECS Optimized AMIs containing ECS Agent 1.44.4 here: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-optimized_AMI.html
